imageroot/actions/alter-group/50alter_group

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (C) 2021 Nethesis S.r.l.
+# http://www.nethesis.it - nethserver@nethesis.it
+#
+# This script is part of NethServer.
+#
+# NethServer is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or any later version.
+#
+# NethServer is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NethServer.  If not, see COPYING.
+#
+
+import agent
+import json
+import sys
+import subprocess
+import os
+
+request = json.load(sys.stdin)
+group = request['group']
+
+sambatool_cmd = ['podman', 'exec', os.environ['MODULE_ID'], 'samba-tool']
+
+if 'users' in request:
```

Comment:
If `users` is not present, should the group emptied?

Reply:
All attributes are optional (apart from `group`, the identifier). If an attribute is missing no change occurs. To empty a group set `users:[]`